thunx-integration-tests/api-integration-tests/src/test/java/com/contentgrid/thunx/example/demo/ThunxDemoApplicationTests.java

```diff
@@ -53,7 +52,7 @@
 
 @Transactional
 @AutoConfigureMockMvc(printOnlyOnFailure = false)
-@SpringBootTest(classes = InvoicingApplication.class)
+@SpringBootTest(classes = InvoicingApplication.class, properties = "contentgrid.thunx.abac.source=header")
```

Comment:
```suggestion
@SpringBootTest(classes = InvoicingApplication.class)
```

This property should not be needed, as thunx should default to this.

Reply:
What if we later update contentgrid-spring to set this property to `none`?